lib/Fmt_ast.ml

```diff
@@ -754,8 +763,15 @@ and fmt_arrow_param c ctx
    [xtyp] should be parenthesized. [constraint_ctx] gives the higher context
    of the expression, i.e. if the expression is part of a `fun`
    expression. *)
+(* I think instead of having a [tydecl_param] argument here, the right thing
```

Comment:
Please clarify who `I` is for the future. And maybe put the words "Jane Street" in the comment so we can find it again? Not sure if we have a convention about such things.